src/NuGetPackageVerifier/CompositeRules/DefaultCompositeRule.cs

```diff
@@ -9,10 +9,8 @@ public class DefaultCompositeRule : IPackageVerifierRule
     {
         IPackageVerifierRule[] _rules = new IPackageVerifierRule[]
```

Comment:
@Eilon we'll keep the two rules around for the time being. This avoids the need to update the verifier json in all the repos at this point.